crates/forge/src/cmd/test/filter.rs

```diff
@@ -42,6 +42,11 @@ pub struct FilterArgs {
     /// Only show coverage for files that do not match the specified regex pattern.
     #[arg(long = "no-match-coverage", visible_alias = "nmco", value_name = "REGEX")]
     pub coverage_pattern_inverse: Option<regex::Regex>,
+
+    /// Qualified test failures from --rerun (contract, test) pairs.
```

Comment:
I don't think we should make this pub arg but rather set (or reuse) the rerun flag?

Reply:
Will remove `pub` in next commit.

Not sure of a better option. The rerun flag is insufficient as a boolean when this is holding internal state that contains the actual list of (contract, test) pairs to rerun.

The flow is:
f--rerun flag → load failure file → populate qualified_failures → use in filtering

As I see it, the alternatives would be:

1. Pass the data separately through multiple function parameters (messier)
2. Global state (bad practice)
3. Reload the file multiple times (inefficient)